# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchTypes.ipdlh
# Commit: 5c3b5a384f2f
# Full Hash: 5c3b5a384f2fb0ee756d2fe024d2cda4fb643aa0
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-12 21:30:33
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1351231 - Support conversation between InternalRequest and IPCInternalRequest. r=dom-worker-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138813
# ==============================================================================

diff -r d8fa8a006948 -r 5c3b5a384f2f dom/fetch/FetchTypes.ipdlh
--- a/dom/fetch/FetchTypes.ipdlh	Thu Jan 12 15:09:44 2023 +0000
+++ b/dom/fetch/FetchTypes.ipdlh	Thu Jan 12 15:09:44 2023 +0000
@@ -18,6 +18,7 @@
 using ResponseType from "mozilla/dom/ResponseBinding.h";
 using struct nsID from "nsID.h";
 using FetchDriverObserver::EndReason from "mozilla/dom/FetchDriver.h";
+using nsILoadInfo::CrossOriginEmbedderPolicy from "nsILoadInfo.h";
 [RefCounted] using class mozilla::RemoteLazyInputStream from "mozilla/RemoteLazyInputStream.h";
 [RefCounted] using class nsITransportSecurityInfo from "nsITransportSecurityInfo.h";
 
@@ -59,6 +60,7 @@
   uint32_t contentPolicyType;
   nsString referrer;
   ReferrerPolicy referrerPolicy;
+  ReferrerPolicy environmentReferrerPolicy;
   RequestMode requestMode;
   RequestCredentials requestCredentials;
   RequestCache cacheMode;
@@ -70,6 +72,7 @@
   uint32_t interceptionContentPolicyType;
   RedirectHistoryEntryInfo[] interceptionRedirectChain;
   bool interceptionFromThirdParty;
+  CrossOriginEmbedderPolicy embedderPolicy;
 };
 
 struct InternalResponseMetadata {